src/garage/torch/modules/gaussian_mlp_module.py

```diff
@@ -0,0 +1,231 @@
+        self._hidden_w_init = hidden_w_init
+        self._hidden_b_init = hidden_b_init
+        self._output_nonlinearity = output_nonlinearity
+        self._output_w_init = output_w_init
+        self._output_b_init = output_b_init
+        self._layer_normalization = layer_normalization
+
+        self._init_std_param = torch.Tensor([init_std])
+        if self._learn_std:
+            self._init_std = torch.nn.Parameter(self._init_std_param)
+
+        self._min_std_param, self._max_std_param = None, None
+        if min_std:
+            min_std = torch.tensor(min_std, dtype=torch.float)
+        if max_std:
+            max_std = torch.tensor(max_std, dtype=torch.float)
+
+        if self._std_parameterization == 'exp':
+            self._init_std_param = self._init_std_param.log()
+            self._min_std_param = min_std.log() if min_std else None
+            self._max_std_param = max_std.log() if max_std else None
+
+        elif self._std_parameterization == 'softplus':
+            self._init_std_param = self._init_std_param.exp().add(-1).log().log()
+            self._min_std_param = min_std.exp().add(-1).log().log() if min_std else None
+            self._max_std_param = max_std.exp().add(-1).log().log() if max_std else None
+
+        else:
+            raise NotImplementedError
+
+        if std_share_network:
+            self.shared_mean_log_std_network = self._get_shared_mean_std_network()
+        else:
+            self.mean_module = self._get_mean_network()
+
+            if adaptive_std:
+                self.adaptive_log_std_module = self._get_adaptive_std_network()
+
+    def _get_shared_mean_std_network(self):
+        return MLPModule(
+            input_dim=self._input_dim,
+            output_dim=self._action_dim * 2,
+            hidden_sizes=self._hidden_sizes,
+            hidden_nonlinearity=self._hidden_nonlinearity,
+            hidden_w_init=self._hidden_w_init,
+            hidden_b_init=self._hidden_b_init,
+            output_nonlinearity=self._output_nonlinearity,
+            output_w_init=self._output_w_init,
+            output_b_init=self._init_shared_b,
+            layer_normalization=self._layer_normalization
+        )
+
+    def _init_shared_b(self, b):
+        nn.init.zeros_(b[..., :self._action_dim])
+        nn.init.constant_(b[..., self._action_dim:], self._init_std_param.item())
+
+    def _get_mean_network(self):
+        return MLPModule(
+            input_dim=self._input_dim,
+            output_dim=self._action_dim,
+            hidden_sizes=self._hidden_sizes,
+            hidden_nonlinearity=self._hidden_nonlinearity,
+            hidden_w_init=self._hidden_w_init,
+            hidden_b_init=self._hidden_b_init,
+            output_nonlinearity=self._output_nonlinearity,
+            output_w_init=self._output_w_init,
+            output_b_init=self._output_b_init,
+            layer_normalization=self._layer_normalization)
+
+    def _get_adaptive_std_network(self):
```

Comment:
`get` is usually used for properties, perhaps you can use the verb `build` or `make`?

Reply:
fixed